executor/instruction/has_reverse_executor.rs

```diff
@@ -116,6 +104,25 @@ impl HasReverseExecutor {
         context: &ExecutionContext<impl ReadableSnapshot + 'static>,
         row: MaybeOwnedRow<'_>,
     ) -> Result<TupleIterator, ConceptReadError> {
+        if self.iterate_mode.is_inverted() && self.attribute_cache.get().is_none() {
+            // TODO: this will only work if the range is actually not dependent on the Row... needs special API
+
+            // one-off initialisation of the cache of constants
+            let value_range =
+                self.checker.value_range_for(context, None, self.has.attribute().as_variable().unwrap())?;
```

Comment:
This is missing from `HasExecutor`'s `BoundFrom` case

Reply:
Yeah i'm aware, thanks